oat-sa/browserslist-app-tao
# Feature/ADF-1419/Upgrade the CI action to Node 18

### Related to: https://oat-sa.atlassian.net/browse/ADF-1419

### Summary

Upgrade the CI action to Node 18

### Details

Set Node.js to version 18 in the action manifest.

### How to test

- checkout the branch
- make sure to have node 18
- `npm ci`
- `npx browserslist@latest --update-db`
- `npm run build-api`
- check the changed files, it should be something like this:
```
diff --git a/package-lock.json b/package-lock.json
index 1ab22fc..7a743ab 100644
--- a/package-lock.json
+++ b/package-lock.json
@@ -5616,9 +5616,9 @@
             }
         },
         "node_modules/caniuse-lite": {
-            "version": "1.0.30001478",
-            "resolved": "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001478.tgz",
-            "integrity": "sha512-gMhDyXGItTHipJj2ApIvR+iVB5hd0KP3svMWWXDvZOmjzJJassGLMfxRkQCSYgGd2gtdL/ReeiyvMSFD1Ss6Mw==",
+            "version": "1.0.30001487",
+            "resolved": "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001487.tgz",
+            "integrity": "sha512-83564Z3yWGqXsh2vaH/mhXfEM0wX+NlBCm1jYHOb97TrTWJEmPTccZgeLTPBUUb0PNVo+oomb7wkimZBIERClA==",
             "funding": [
                 {
                     "type": "opencollective",
diff --git a/public/api.json b/public/api.json
index c946005..fe9d645 100644
--- a/public/api.json
+++ b/public/api.json
@@ -1 +1 @@
-[{"browser":"Chrome","os":"","versions":["111","112"],"device":"desktop"},{"browser":"Edge","os":"","versions":["110","111"],"device":"desktop"},{"browser":"Firefox","os":"","versions":["110","111"],"device":"desktop"},{"browser":"Safari","os":"","versions":[15,16],"device":"desktop"},{"browser":"Chrome","os":"Android","versions":["111"],"device":"mobile"},{"browser":"Firefox","os":"Android","versions":["110"],"device":"mobile"},{"browser":"Safari","os":"iOS","versions":[15,16],"device":"mobile"}]
\ No newline at end of file
+[{"browser":"Chrome","os":"","versions":["112","113"],"device":"desktop"},{"browser":"Edge","os":"","versions":["112","113"],"device":"desktop"},{"browser":"Firefox","os":"","versions":["112","113"],"device":"desktop"},{"browser":"Safari","os":"","versions":[15,16],"device":"desktop"},{"browser":"Chrome","os":"Android","versions":["113"],"device":"mobile"},{"browser":"Firefox","os":"Android","versions":["113"],"device":"mobile"},{"browser":"Safari","os":"iOS","versions":[15,16],"device":"mobile"}]
\ No newline at end of file
```
